Backend/dev/migrations/dynamic-offer-driver-app/0139-search-request-and-search-step-tables.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE atlas_driver_offer_bpp.search_request RENAME TO search_step;
```

Comment:
No backward compaitiblity.

Reply:
> No backward compaitiblity.

I don't really understand why you leave this comment for this migration and not for EPs, but it's nearly impossible to make this PR backward compatible. I can try, but it will take a lot of time and i am not really sure about a result.